docs/sources/next/javascript-api/jslib/http-instrumentation-tempo/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "http instrument tempo"
+descriptiontion: "k6 tempo instrumentation API"
+weight: 04
+weight: 04
+---
+
+# Tempo
+
+
+With this jslib, you can _instrument_ HTTP requests so that they emit traces as the test runs.
+
+## About trace contexts
```

Comment:
While trying to edit the old docs to these ones ... I pretty much decide that the `tempo` name is ... pretty bad and it probably should be `tracing` instead.

Reply:
In the spirit of getting things merge, we'll keep it name as-is for now, and let's reevaluate at a future point in time if we deem it necessary 👍🏻